Dockerfile

```diff
@@ -33,9 +33,9 @@ RUN apk add --update --no-cache \
     /.cache
 
 COPY --chmod=755 entry.sh /entry.sh
-COPY --link requirements.txt /
+COPY requirements.txt /
 
 RUN python3 -m pip install --no-cache -Ur requirements.txt
 RUN borgmatic --bash-completion > /usr/share/bash-completion/completions/borgmatic && echo "source /etc/profile.d/bash_completion.sh" > /root/.bashrc
 
-CMD ["/entry.sh"]
+ENTRYPOINT ["/entry.sh"]
```

Comment:
The reason this is `CMD` and not `ENTRYPOINT` is so people can `docker run -it --rm b3vis/docker-borgmatic bash` and get a shell if needed

Reply:
I see this is to support the binary use case - resolving